.github/workflows/ci.yml

```diff
@@ -29,4 +29,4 @@ jobs:
 
     - uses: actions/checkout@v2.3.4
 
-    - run: ./tools/ci.sh
+    - run: ./tools/ci.sh
```

Comment:
@Leeyoungone This might be because of a new line was either removed or added. Please if possible undo the change so that the PR is only changing the docs (I'm nit picking, apologies)